src/sql/workbench/services/notebook/browser/sql/sqlSessionManager.ts

```diff
@@ -426,7 +430,7 @@ export class SQLFuture extends Disposable implements FutureInternal {
 			channel: 'shell',
 			type: 'execute_reply',
 			content: {
-				status: 'ok',
+				status: this._errorOccurred && this._stopOnError ? 'error' : 'ok',
```

Comment:
Does the err param not have this information? It seems convoluted to have to track this at the class level.